Homework_6/commentify/frontend/src/Api/api.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "http://localhost:5000/api",
```

Comment:
But I asked chatGPT where to put env file in frontend, answer was not necessary to have env file in front, because it might contain sensitive information. But I created enc file in backend